ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml

```diff
@@ -9,16 +9,59 @@ spec:
     - name: pyxis_url
     - name: cert_project_id
     - name: bundle_version
+    - name: operator_name
   results:
     - name: preflight_results_exists
+    - name: test_result_id
+    - name: test_logs_id
   workspaces:
     - name: results
   steps:
-    - name: get-ci-results
+    - name: get-results
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
+      workingDir: $(workspaces.results.path)
+      script: |
+        #! /usr/bin/env bash
+
+        # Send request to Pyxis to get the logs and test results.
+        # If they exists, store them in the workspace
+
+        echo "Downloading the artifacts"
+
+        download-artifacts \
+          --pyxis-url "$(params.pyxis_url)" \
+          --cert-project-id "$(params.cert_project_id)" \
+          --certification-hash "$(params.md5sum)" \
+          --operator-package-version "$(params.bundle_version)" \
+          --operator-name "$(params.operator_name)" \
+          --cert-path /cert/tls.crt \
```

Comment:
There will be multiple different tasks that need the cert and key. The paths should probably be defined at the pipeline level then passed into the task, instead of hardcoding it in each task.